src/content/community/conferences.md

```diff
@@ -15,16 +15,29 @@ March 20 - 21, 2025. In-person in Paris, France (hybrid event)
 
 [Website](https://react.paris/) - [Twitter](https://x.com/BeJS_)
 
+
```

Comment:
seems like there were some extra line feeds that were added. can you please remove these and keep the spaces consistent as older confs? Thanks!